storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -869,84 +912,27 @@ void Archived_Redo_Log_Monitor::thread_func() {
   my_thread_end();
 }
 
-/* XB30 link to create_log_file */
-static dberr_t open_or_create_log_file(bool *log_file_created, ulint i,
-                                       fil_space_t **log_space) {
-  bool ret;
-  os_offset_t size;
-  char name[10000];
-  ulint dirnamelen;
-
-  *log_file_created = false;
-
-  Fil_path::normalize(srv_log_group_home_dir);
-
-  dirnamelen = strlen(srv_log_group_home_dir);
-  ut_a(dirnamelen < (sizeof name) - 10 - sizeof "ib_logfile");
-  memcpy(name, srv_log_group_home_dir, dirnamelen);
-
-  /* Add a path separator if needed. */
-  if (dirnamelen && name[dirnamelen - 1] != OS_PATH_SEPARATOR) {
-    name[dirnamelen++] = OS_PATH_SEPARATOR;
-  }
-
-  sprintf(name + dirnamelen, "%s%ld", "ib_logfile", i);
-
-  pfs_os_file_t file = os_file_create(0, name, OS_FILE_OPEN, OS_FILE_NORMAL,
-                                      OS_LOG_FILE, true, &ret);
-  if (!ret) {
-    xb::error() << "error in opening " << name;
-
-    return (DB_ERROR);
-  }
-
-  size = os_file_get_size(file);
-
-  if (size != srv_log_file_size) {
-    xb::error() << "log file " << name << " is of different size " << size
-                << " bytes than specified in the .cnf file "
-                << srv_log_file_size << " bytes!";
-
-    return (DB_ERROR);
-  }
-
-  ret = os_file_close(file);
-  ut_a(ret);
-
-  if (i == 0) {
-    /* Create in memory the file space object
-    which is for this log group */
-
-    *log_space = fil_space_create(name, dict_sys_t::s_log_space_id,
-                                  fsp_flags_set_page_size(0, univ_page_size),
-                                  FIL_TYPE_TEMPORARY);
-  }
-
-  ut_a(*log_space != NULL);
-  ut_ad(fil_validate());
-
-  ut_a(fil_node_create(name, srv_log_file_size / univ_page_size.physical(),
-                       *log_space, false, false) != NULL);
-
-  return (DB_SUCCESS);
-}
-
 bool Redo_Log_Data_Manager::init() {
+  /*XB30 srv_redo_log_capacity */
+  srv_redo_log_capacity = srv_redo_log_capacity_used = 1024 * 1024 * 1024;
```

Comment:
can you please what happens when you remove this? Because this was fixed for prepare.

In this case, it is backup and we just need to read the redo logs.

Reply:
I have replaced it with 
srv_redo_log_capacity = srv_redo_log_capacity_used = 0; Is it fine?

Reply:
Please check if it works fine. During backup, we initialize redo subsystem only for LSN parsing/etc. We don't generate redo at all from backup

Reply:
basic test is fine. But will analyze jenkin as well.

Reply:
Ok, marking it as resolved